monoio/Cargo.toml

```diff
@@ -59,6 +59,8 @@ async-cancel = []
 # enanle zero copy(enable SOCK_ZEROCOPY + MSG_ZEROCOPY flag)
 # WARNING: this feature may cause performance degradation
 zero-copy = []
+# splice op(require kernel 5.7+)
```

Comment:
Why we need a feature flag for `splice`, maybe we can assert the kernel version.

Reply:
It will make us require higher kernel version(5.4->5.7).

Reply:
Maybe in the future the feature will be delete and make it as default.